packages/components/src/components/ResizableBox/ResizableBox.tsx

```diff
@@ -272,7 +274,12 @@ export const ResizableBox = ({
             }
         };
 
-        document.onmouseup = () => setIsResizing(false);
+        document.onmouseup = () => {
+            setIsResizing(false);
+            if (onSaveWidth) {
+                onSaveWidth(newWidth);
```

Comment:
That's new info for me. But ok. Will add it. Do we want to use 2 different props for it or do it within one? Or doesn't really matter?

Reply:
Makes sense. Just.. it's gonna be unused (and hence unnecessary). Because for the sidebar, it doesn't make sense to save its height to SuiteSettings or doing anything else with the height on mouse up that isn't already implemented in ResizableBox.

Reply:
On the other hand it is not the best practice to add unnecessary functionality just in case it might be used sometimes, so we should be a bit careful in such situations as now. Fortunately this is a very small change and maybe possibly useful in the future so I don't mind.

Reply:
This component already handles `height` resize which we are not using. 

Either:
1. Lets remove height logic (its symmetrical logic to width logic) and its very easy to add later (just by copy-pasting the with
2. Or lets threat it with same level of completeness as `width` and lets have the `onXXXX` handler for both.

As `width` and `height` never changes together (we don't have corner drag implemented, and I assume we dont plan to) -> therefore I suggest to have 2 callbacks: `onWidthXXX` and `onHeightXXX`



Subjectively I prefer option 2.